packages/components/src/internal/util/helpLinks.tsx

```diff
@@ -41,15 +41,33 @@ export const SAMPLE_ALIQUOT_TOPIC = 'aliquot';
 
 export const UNIQUE_IDS_TOPIC = 'uniqueStorageIds';
 
-export function getHelpLink(topic: string): string {
-    return getServerContext().helpLinkPrefix + topic;
+export function getHelpLink(topic: string, referrer = 'inPage'): string {
```

Comment:
nit: Just curious, are there an explicit set of expected valid values for `referrer`?

Reply:
Yes there is. I have added a `HELP_LINK_REFERRER` enum